### PR TITLE
Update lock file after logging changes in plugins

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

**Comment by suyograo** [comment]
```
suyog@machine:~/ws/elastic/logstash/build/logstash-5.0.0-beta1-SNAPSHOT (update_lock_file2)$ bin/logstash-plugin list --verbose beats
logstash-input-beats (3.1.4)
```

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -245,10 +245,10 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       nokogiri
       xml-simple
-    logstash-input-beats (3.1.4)
+    logstash-input-beats (3.1.4-java)
       concurrent-ruby (>= 0.9.2, <= 1.0.0)
       jar-dependencies (~> 0.3.4)
-      logstash-codec-multiline (~> 3.0)
+      logstash-codec-multiline (>= 2.0.5)
```
Comment (talevy) [inline Gemfile.jruby-1.9.lock]: I guess this applies to input-beats, not these changes, but, why the rather large change on dependency restrictions here?

Comment (suyograo) [inline Gemfile.jruby-1.9.lock]: I'm not sure actually. I never hand-edit this file, so not sure how to control it. The gemspec didn't have any changes of this sort.

Comment (talevy) [inline Gemfile.jruby-1.9.lock]: ah, https://github.com/logstash-plugins/logstash-input-beats/commit/1a0b0b217ac98595fee28a4b2f3a34437e763e5e

LS 2.4 and the plugin api 👍

**Comment by ph** [comment]
LGTM

**Pull request merged**